diff_tf:
  ros__parameters:
    rate: 10
    ticks_meter: 320
    base_width: 0.245
    base_frame_id: base_link
    odom_frame_id: odom
    encoder_min: -32768
    encoder_max: 32768
    # wheel_low_wrap 
    # wheel_high_wrap

twist_to_motors:
  ros__parameters:
    base_width: 0.1
    rate: 50
    timeout_ticks: 2

virtual_joystick:
  ros__parameters:
    publish_rate: 10
    x_min: -1.5
    x_max: 1.5
    r_min: 10.0
    r_max: 10.0